packages/config/src/chainConfig/presets/mainnet.ts

```diff
@@ -36,6 +36,9 @@ export const chainConfig: IChainConfig = {
   // Bellatrix
   BELLATRIX_FORK_VERSION: b("0x02000000"),
   BELLATRIX_FORK_EPOCH: 144896, // Sept 6, 2022, 11:34:47am UTC
+  // Capella
+  CAPELLA_FORK_VERSION: b("0x03000000"),
+  CAPELLA_FORK_EPOCH: Infinity,
   // Sharding
   SHARDING_FORK_VERSION: b("0x03000000"),
```

Comment:
Should not have the same fork_version

Reply:
should i rename them to 4844 something, thats what the new spec calls them

Reply:
rebased/removed